libmscore/style.cpp

```diff
@@ -403,6 +402,7 @@ static const StyleType styleTypes[] {
       { Sid::minEmptyMeasures,        "minEmptyMeasures",        QVariant(2) },
       { Sid::minMMRestWidth,          "minMMRestWidth",          Spatium(4) },
       { Sid::mmRestNumberPos,         "mmRestNumberPos",         Spatium(-1.5) },
+      { Sid::multiMeasureRestMargin,  "multiMeasureRestMargin",  Spatium(1.2)  },
```

Comment:
isn't that new functionality?

Reply:
Nope, that line'd been way above (line 187) but got moved down to line 405.

Reply:
Not really necessary, I guess.